changelogs/fragments/1476-add-redis6x-cache-parameter-group-family

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - elasticache_parameter_group - add new Redis cache parameter group family on the module input choices (https://github.com/ansible-collections/community.aws/pull/1476).
```

Comment:
```suggestion
  - elasticache_parameter_group - add ``redis6.x`` group family on the module input choices (https://github.com/ansible-collections/community.aws/pull/1476).
```

Otherwise lgtm